.github/workflows/test.yml

```diff
@@ -24,15 +24,26 @@ jobs:
     - name: Upgrade pip
       run: |
         python -m pip install --upgrade pip
+
     - name: Install Poetry and project dependencies
       run: |
         pip install poetry
         poetry install
+
     - name: Run Pyright
       run: poetry run pyright
 
     - name: Run Pytest
       run: poetry run pytest
 
     - name: Run MyPy
-      run: poetry run mypy pandas-stubs tests
\ No newline at end of file
+      run: poetry run mypy pandas-stubs tests
+
+    - name: Install wheel
+      run: poetry run install_wheel
+
+    - name: Run Pyright against dist
```

Comment:
So my only concern here is that pyright and mypy might be looking at the source distribution for `pandas-stubs`, instead of what is installed.  That's why I had a `rm -rf typings/pandas` to make sure that they weren't looking there once the wheel was installed.

So can you do something similar here - could `rm` or `mv` the directory for these two tests.

Reply:
Sure, Just a second